custom_components/awox/__init__.py

```diff
@@ -35,7 +35,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     # Make `mesh` accessible for all platforms
     hass.data[DOMAIN][entry.entry_id] = mesh
 
-    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
+    # DEPRECATED: hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

Comment:
Tnx for the PR

This line can be removed. No need to leave the depredation comment. That's what version control is for